src/main/java/bio/terra/cli/context/WorkspaceContext.java

```diff
@@ -33,6 +33,10 @@
   // map of cloud resources for this workspace (name -> WSM resource description object)
   public Map<String, ResourceDescription> resources;
 
```

Comment:
Would it be a lot of trouble to add a `terra config` property to override this, so we have a workaround without a recompile if somebody goes over the edge?

Reply:
Great idea. I'll add a new config property for it.

Reply:
Added `terra config set resources` and `terra config get-value resources`. Screenshot included in PR description.